source/includes/common/common-deploy.rst

```diff
@@ -32,7 +32,25 @@ The following example provides a starting environment file:
 
    #MINIO_SERVER_URL="http://minio.example.net:9000"
 
-Include any other environment variables as required for your local deployment.
+Include any other environment variables as required for your deployment.
+
+.. versionadded:: Server RELEASE.2024-03-03T17-50-39Z
+
+   You can have MinIO automatically generate root credentials if all of the following are true:
```

Comment:
"Can" ? Seems like "will" if the prerequisites are met. Or is there a way to disable the automagical creds if you didn't want them? (Whether you should or not might be another story.)

Reply:
If the conditions are meet, MinIO will always generate the credentials auto.
Of course, you can simply opt-out by setting the credentials yourself.

Reply:
Rephrased